--- PUCLogin.jsp.orig	2014-07-18 14:43:04.000000000 +0200
+++ PUCLogin.jsp	2014-07-18 14:43:52.000000000 +0200
@@ -175,55 +175,8 @@
               <input type="hidden" name="locale" value="en_US">
             </div>
           </div>
-          <div id="eval-users-toggle-container">
-            <%
-              if (showUsers) {
-            %>
-            <div id="eval-users-toggle" onClick="toggleEvalPanel()">
-              <div><%=Messages.getInstance().getString("UI.PUC.LOGIN.EVAL_LOGIN")%></div>
-                <div id="eval-arrow" class="closed"></div>
-            </div>
-
-            <%
-            } else {
-            %>
-            &nbsp;
-            <%
-              }
-            %>
-          </div>
         </form>
       </div>
-
-      <div class="row-fluid">
-        <div id="evaluationPanel" class="span10 row-fluid">
-          <div id="role-admin-panel" class="span6 well">
-            <div class="login-role"><%=Messages.getInstance().getString("UI.PUC.LOGIN.ADMIN_USER")%></div>
-            <div class="row-fluid">
-              <div class="span6 login-label"><%=Messages.getInstance().getString("UI.PUC.LOGIN.USERNAME")%></div>
-              <div class="span6 login-value">Admin</div>
-            </div>
-            <div class="row-fluid">
-              <div class="span6 login-label"><%=Messages.getInstance().getString("UI.PUC.LOGIN.PASSWORD")%></div>
-              <div class="span6 login-value">password</div>
-            </div>
-            <button class="btn" onClick="loginAs('Admin', 'password');"><%=Messages.getInstance().getString("UI.PUC.LOGIN.GO")%></button>
-          </div>
-          <div id="role-business-user-panel" class="span6 well ">
-            <div class="login-role"><%=Messages.getInstance().getString("UI.PUC.LOGIN.BUSINESS_USER")%></div>
-            <div class="row-fluid">
-              <div class="span6 login-label"><%=Messages.getInstance().getString("UI.PUC.LOGIN.USERNAME")%></div>
-              <div class="span6 login-value">Suzy</div>
-            </div>
-            <div class="row-fluid">
-              <div class="span6 login-label"><%=Messages.getInstance().getString("UI.PUC.LOGIN.PASSWORD")%></div>
-              <div class="span6 login-value">password</div>
-            </div>
-            <button class="btn" onClick="loginAs('Suzy', 'password');"><%=Messages.getInstance().getString("UI.PUC.LOGIN.GO")%></button>
-        </div>
-      </div>
-      </div>
-
     </div>
   </div>
   <div id="login-footer-wrapper">
